Main.java

```diff
@@ -9,7 +11,26 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
-    
+    // Create a scanner for user input
+    Scanner input = new Scanner(System.in);
+
+    //Gets the users first interger and prints it on the screen
+    System.out.println("Please enter the first interger:");
+    int a = input.nextInt();
+    
+    //Gets the users second interger and prints it on the screen
+    System.out.println("Please enter the second interger:");
+    int b = input.nextInt();
+
+    //Checks to see which number is larger
+    if (a > b) { 
+    System.out.println("The first number is larger");
+    } else if (b > a) {
+    System.out.println("The second number is larger");
+    } else if (a == b) {
+    System.out.println("Both of the intergers are equal");
+    }
```

Comment:
Watch your indentation. If there is code inside of an if statement, it should be TABBED over an extra bit to show it is inside of the if-statement. Otherwise, good work. Just make sure you are trying to match the sample output as closely as you can.